tools/shared.py

```diff
@@ -1317,11 +1317,19 @@ def __getitem__(self, key):
     return self.instance()[key]
 
 
-Settings = SettingsManager()
+def verify_settings():
+  if Settings.WASM_BACKEND and not Settings.WASM:
+    logging.error('emcc: WASM_BACKEND is not compatible with asmjs (WASM=0)')
```

Comment:
Land #6866 first and use `exit_with_error` here I think

Reply:
Yup